src/qt/macnotificationhandler.mm

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2011-2013 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include "macnotificationhandler.h"
+
+#undef slots
+#import <objc/runtime.h>
+#include <Cocoa/Cocoa.h>
+
+// Add an obj-c category (extension) to return the expected bundle identifier
+@implementation NSBundle(returnCorrectIdentifier)
+- (NSString *)__bundleIdentifier
+{
+    if (self == [NSBundle mainBundle]) {
+        return @"org.hyperstake.HyperStake-Qt";
```

Comment:
@konez2k it looks like we should change this to `org.buzzcoin.BuzzCoin-Qt`.

Reply:
Makefile, Makefile.Release, Makefile.Debug is created by qmake on the .pro file, they are local files, only for build purposes and not needed on repository cause they are different per platform.

Same applies to the removed folders, which are recreated automatically when you build.